src/main/java/com/endava/soa_5ed/task1_basics/Ex4.java

```diff
@@ -0,0 +1,41 @@
+package com.endava.soa_5ed.task1_basics;
+
+import java.util.Scanner;
+
+public class Ex4 {
+//    Write 3 methods that calculates sum of numbers from 0 to a given number.
+//            (1) Use for loop
+//            (2) Use while loop
+//            (3) Use do while loop
+
+    //1st method - for loop
+    int calc1(int num){
+        int sum=0;
+        for(int i=0; i<=num; i++){
+            sum+=i;
+        }
+        return sum;
+    }
+    //2nd method - while loop
+    int calc2(int num){
+        int sum=0;
+        int i=0;
```

Comment:
you could start iterating with 1, as now in the first iteration you would add 0 to the sum, which does not change the final value of sum. If you change to 1, the first iteration would be meaningful and the code will be a little more efficient.

Reply:
done.